packages/form/src/components/SwitchButtonField/index.tsx

```diff
@@ -0,0 +1,55 @@
+import type { FieldPath, FieldValues } from '@ultraviolet/form'
+import { SwitchButton } from '@ultraviolet/ui'
+import type { ComponentProps } from 'react'
+import { useController } from 'react-hook-form'
+import type { BaseFieldProps } from '../../types'
+
+type SwitchButtonFieldProps<
+  TFieldValues extends FieldValues,
+  TFieldName extends FieldPath<TFieldValues>,
+> = BaseFieldProps<TFieldValues, TFieldName> &
+  Omit<ComponentProps<typeof SwitchButton>, 'value' | 'onChange' | 'name'> & {
+    name: FieldPath<TFieldValues>
+  }
```

Comment:
can We add the original onChange like for onBlur ? if we want to upfate other field based on this onChange ?